dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -405,13 +413,37 @@ object LagomPlugin extends AutoPlugin {
   }
 
   private def runAllMicroservicesTask: Initialize[Task[Unit]] = Def.taskDyn {
+
     val projects = microservicesProjects.value
-    val filter = ScopeFilter(inProjects(projects: _*))
+    val filter: ScopeFilter = ScopeFilter(inProjects(projects: _*))
+    // check if application.conf is present in service
+    val serviceResourceDirectories = ((unmanagedResourceDirectories in Runtime) ?? Nil).all(filter)
+    //Verifying the presence of the required application.conf file
+    val verifyApplicationConfTask = Def.task {
+
+      val servicesPaths: Seq[File] = serviceResourceDirectories.value.flatten
+
+      val (validProject, notValidProject) = servicesPaths.partition(f => {
+        val r: PathFinder = {
+          f.get * "application.conf"
+        }
+        r.get.nonEmpty
+      })
+      //displays all projects that not contain a application.conf file
+      if (notValidProject.nonEmpty) {
+        notValidProject foreach (f => {
+          val log = state.value.log
+          log.error("File application.conf has been missed in the following service >>> " + f.project)
```

Comment:
It would be better if the message was something like:

`log.error(s"File application.conf is missing in project ${(name in f.project).value}")`

Also, I'm now realising that a user, after reading this message, will likely just create an empty application.conf. That would be a problem because the service will fail to start, with the same exception reported in the ticket this PR attempts to provide a fix for. So, after all, maybe this is not the right approach to fix the problem... (?)